backend/django/core/views/api_annotate.py

```diff
@@ -427,29 +428,50 @@ def data_unlabeled_table(request, project_pk):
     Returns:
         data: a list of data information
     """
-    project = Project.objects.get(pk=project_pk)
+    unlabeled_data = get_unlabeled_data(project_pk)
+    data = []
```

Comment:
Same as with the `search_data_unlabeled_table` but simpler. I think we can get this effect faster by moving the serialization to bulk with something like this.

```
    unlabeled_data = get_unlabeled_data(project_pk)[:50]
    serialized_data = DataSerializer(unlabeled_data, many=True).data
    data = [{
        "Text": d["text"],
        "metadata": d["metadata"],
        "ID": d["pk"],
    } for d in serialized_data]
    return Response({"data": data})
```